package.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "Football_teams",
+  "version": "1.0.0",
+  "description": "### About :",
+  "main": "index.js",
+  "scripts": {
+    "test": "node src/test.js",
+    "dev": "nodemon src/server.js",
+    "eslint": "eslint src/*",
```

Comment:
You should have a start script, 
`"start" : "node src/server.js"`

Reply:
@ghassanmas  we add this to script